webpack/assets/javascripts/react_app/components/common/forms/__snapshots__/Actions.test.js.snap

```diff
@@ -12,7 +12,7 @@ exports[`actions buttons could be disabled 1`] = `
       block={false}
       bsClass="btn"
       bsStyle="primary"
-      disabled={true}
+      disabled={false}
```

Comment:
is this a bug?

Reply:
It keeps the 'Cancel' button enabled even if user haven't touched the form.